examples/js/loaders/STLLoader.js

```diff
@@ -164,34 +163,28 @@ THREE.STLLoader.prototype = {
 
 				var vertexstart = start + i * 12;
 
-				vertices[ offset ] = reader.getFloat32( vertexstart, true );
-				vertices[ offset + 1 ] = reader.getFloat32( vertexstart + 4, true );
-				vertices[ offset + 2 ] = reader.getFloat32( vertexstart + 8, true );
+				vertices.push( reader.getFloat32( vertexstart, true ) );
+				vertices.push( reader.getFloat32( vertexstart + 4, true ) );
+				vertices.push( reader.getFloat32( vertexstart + 8, true ) );
 
-				normals[ offset ] = normalX;
-				normals[ offset + 1 ] = normalY;
-				normals[ offset + 2 ] = normalZ;
+				normals.push( normalX, normalY, normalZ );
 
 				if ( hasColors ) {
 
-					colors[ offset ] = r;
-					colors[ offset + 1 ] = g;
-					colors[ offset + 2 ] = b;
+					colors.push( r, g, b );
 
 				}
 
-				offset += 3;
-
 			}
 
 		}
 
-		geometry.addAttribute( 'position', new THREE.BufferAttribute( vertices, 3 ) );
```

Comment:
I just find it weird that we have `Float32Attribute` (or `Float32BufferAttribute`) and `InstancedBufferAttribute` at the same time. There is something wrong in the API design there.

It's likely that `InstancedBufferAttribute` is the bad one here.